storage/innobase/innodb.cmake

```diff
@@ -36,7 +36,7 @@ MYSQL_CHECK_NUMA()
 
 INCLUDE(${MYSQL_CMAKE_SCRIPT_DIR}/compile_flags.cmake)
 
-IF(CMAKE_CROSSCOMPILING)
+IF(CMAKE_CROSSCOMPILING AND NOT DEFINED CMAKE_CROSSCOMPILING_EMULATOR)
```

Comment:
this one is ok as is because of ELSE condition.

Reply:
There are you right, I deleted it there thank you!